homeassistant/components/automation/services.yaml

```diff
@@ -27,6 +27,9 @@ trigger:
     entity_id:
       description: Name of the automation to trigger.
       example: 'automation.notify_home'
+    skip_condition:
+      description: Whether or not the condition will be skipped (defaults to True).
+      example: True
```

Comment:
Actually, shouldn't this be "true" (lowercase)? That's how it is for the `media_player.volume_mute` service.

Reply:
It doesn't matter. YAML detects a boolean anyways. Please correct me if I'm wrong.

https://yaml.org/type/bool.html

Reply:
You're right, True and False work.